### PR TITLE
Update API endpoint from api.raygun.io to api.raygun.com

### PR DESCRIPTION
**Pull request by turtlespin**
This PR changes the default endpoint from api.raygun.io to api.raygun.com as .com is now the recommended endpoint.

### CODE REVIEW

File: src/raygun.js [hunk 1]
```diff
@@ -56,7 +56,7 @@ var raygunFactory = function(window, $, undefined) {
     _groupingKeyCallback,
     _beforeXHRCallback,
     _afterSendCallback,
-    _raygunApiUrl = 'https://api.raygun.io',
```
Comment (darcythomas) [inline src/raygun.js:59]: Could this be a breaking change?

Comment (Olwiba) [reply src/raygun.js:59]: Perhaps it something to check with Denny but I believe for the API supports both api.raygun.io (legacy) and api.raygun.com

**Review by Olwiba — APPROVED** [review]
I think the changes here look good. Please make sure to test this locally to ensure it's OK. 
I spoke with Jasen and we both are in agreeance that that .io/.com domains maintain feature parody (also seeing as JB made this PR, I'm going to say it's allgood)

**Comment by darcythomas** [comment]
This changes were merged in with https://github.com/MindscapeHQ/raygun4js/pull/492

Closing now.

**Pull request closed without merging**